README.md

```diff
@@ -1,5 +1,10 @@
 # jshint loader for webpack
 
+## npm package
+
+``npm install --save jshint-loader jshint``
```

Comment:
Hi @kostasmanionis please have a look [here](https://github.com/webpack/jshint-loader/pull/31), i have updated again and let me know if anything else remaining, i would love to.